### ISSUE TITLE
unable to find opencv2/viz.hpp

### ISSUE THREAD

**Issue by rfsch** [UnaNancyOwen/VelodyneCapture#31]
Hi 

I am running this sample with  a VS 2017 and WIN 10 installation. I used vcpkg. I can only see viz.hpp
in C:\vcpkg\buildtrees\opencv\src\3.4.1-273ea13f6d\modules 

Is there something to specify that viz.hpp shows up in opencv2/ ? 

Thanks!

**Comment by UnaNancyOwen** [comment]
You don't seem to specify an option.
You can install OpenCV with VTK using following command.
For detail options, Please read CONTROL and portfile.cmake of OpenCV port.
```
.\vcpkg install opencv[vtk]
```
Also, Please post to [vcpkg](https://github.com/Microsoft/vcpkg) about how to use vcpkg.

**Comment by rfsch** [comment]
Hi 

thanks again for helping. It works now! I do have another question :) Where can I ask about the pcl library ? I mean I posted some issues on https://github.com/PointCloudLibrary/pcl but got no response...
I tried to post to the http://www.pcl-users.org/mailing_list but the post doesnt show up.. 

Is there a forum where people using pcl help each other ? 

Thanks! and sorry for the off-topic question 

Rupert
